expdj/apps/turk/views.py

```diff
@@ -208,6 +215,15 @@ def preview_hit(request,hid):
         hit =  get_hit(hid,request)
         battery = hit.battery
         context = get_amazon_variables(request)
+
+        missing_batteries, blocking_batteries = battery.check_battery_dependencies(context["worker_id"])
```

Comment:
I think it would help discourage people from accepting a hit that we don't want them to do.